views/pages/index.ejs

```diff
@@ -235,7 +235,7 @@
                     <div id = "scroll-value">&#x1f815</div>
                 </div>
 
-                <div class="text-sm items-center justify-center flex flex-col mb-24">
+                <div class="raleway text-sm items-center justify-center flex flex-col mb-24">
```

Comment:
this is literally a one word change, please make sure you fix these all too : 

- Reduce the `Scroll-to-top` button to smaller size as compared to desktop view.
- Fix the font in Chatbot, it should also be `raleway`

Reply:
Thank you for the feedback I will make the changes.

Reply:
Done with the changes
![image](https://github.com/user-attachments/assets/77520d26-30ce-416d-b3db-8f2e71fdd598)
![image](https://github.com/user-attachments/assets/50257dbc-7da2-48ec-b625-12094ce3435f)